tests/flow/test_stress.py

```diff
@@ -85,18 +90,14 @@ def __del__(self):
 
     # Count number of nodes in the graph
     def test00_stress(self):
-        threads = []
-        for i in range(CLIENT_COUNT):
-            graph = graphs[i]
-            t = threading.Thread(target=query_crud, args=(graph, i))
-            t.setDaemon(True)
-            threads.append(t)
-            t.start()
-
-        # wait for threads to return
-        for i in range(CLIENT_COUNT):
-            t = threads[i]
-            t.join()
+        indexes = range(CLIENT_COUNT)
+        pool = Pool(nodes=CLIENT_COUNT)
+
+        # invoke queries
+        m = pool.amap(query_crud, graphs, indexes)
+
+        # wait for processes to return
+        m.wait()
```

Comment:
switch to `pool.map` which should be synchronized, in which case no need to call `m.wait`